sympy/physics/mechanics/functions.py

```diff
@@ -341,6 +341,45 @@ def potential_energy(*body):
             raise TypeError('*body must have only Particle or RigidBody')
     return pe_sys
 
+def gravity(acceleration, *bodies):
+    """
+    Returns a list of gravity forces given the acceleration
+    due to gravity and any number of particles or rigidbodies.
+
+    Examples
+    ========
+
+>>> from sympy.physics.mechanics import ReferenceFrame, Point, Particle, outer, RigidBody
```

Comment:
The full docstring should be indented four spaces.

Reply:
Sorry about that. I am not able to run bin/doctest for some reason. I am able to run bin/test.
This is the error I am getting 

Traceback (most recent call last):
  File "C:\Python36\lib\site-packages\theano\gof\lazylinker_c.py", line 75, in <module>
    raise ImportError()
ImportError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Python36\lib\site-packages\theano\gof\lazylinker_c.py", line 92, in <module>
    raise ImportError()
ImportError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\pappu\Desktop\Diving_Into_SymPy\sympy4\sympy\sympy\utilities\runtests.py", line 712, in _doctest
    if import_module('theano') is None:
  File "C:\Users\pappu\Desktop\Diving_Into_SymPy\sympy4\sympy\sympy\external\importtools.py", line 133, in import_module
    mod = __import__(module, **__import__kwargs)
  File "C:\Python36\lib\site-packages\theano\__init__.py", line 110, in <module>
    from theano.compile import (
  File "C:\Python36\lib\site-packages\theano\compile\__init__.py", line 12, in <module>
    from theano.compile.mode import *
  File "C:\Python36\lib\site-packages\theano\compile\mode.py", line 11, in <module>
    import theano.gof.vm
  File "C:\Python36\lib\site-packages\theano\gof\vm.py", line 673, in <module>
    from . import lazylinker_c
  File "C:\Python36\lib\site-packages\theano\gof\lazylinker_c.py", line 127, in <module>
    preargs=args)
  File "C:\Python36\lib\site-packages\theano\gof\cmodule.py", line 2359, in compile_str
    (status, compile_stderr.replace('\n', '. ')))
Exception: Compilation failed (return status=1): In file included from C:/mingw/.                  from C:\Users\pappu\AppData\Local\Theano\compiledir_Windows-10-10.0.15063-SP0-Intel64_Family_6_Model_78_Stepping_3_GenuineIntel-3.6.4-32\lazy. C:/mingw/mingw32/lib/gcc/i686-w64-mingw32/7.1.0/include/c++/cmath:1136:11: err.            ^~~~~;ot been declared

You can find the C code in this temporary file: C:\Users\pappu\AppData\Local\Temp\theano_compilation_error_ewxf8ay5